src/pages/deposit/index.tsx

```diff
@@ -0,0 +1,7 @@
+import DepositBar from "@/components/Deposit";
+
+const Deposit = () => {
+  return <DepositBar />;
+};
+
+export default Deposit;
```

Comment:
just wondering, is it necessary to add the deposit to a new page? because it won't be updated frequently and won't display much data.

Reply:
> just wondering, is it necessary to add the deposit to a new page? because it won't be updated frequently and won't display much data.

It have to, the files under the folder <pages> will automatically be treated as a router item. For example, 'pages/deposit/index.tsx' will be able to be accessed by "localhost:3000/deposit"